example_dags/example_transfer_and_return_files.py

```diff
@@ -0,0 +1,39 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+from universal_transfer_operator.datasets.file.base import File
+from universal_transfer_operator.universal_transfer_operator import UniversalTransferOperator
+
+s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk-test")
+gcs_bucket = os.getenv("GCS_BUCKET", "gs://uto-test")
+
+
+def calculate_the_file_transferred(ti):
+    """Return the number of files transferred."""
+    transferred_files = ti.xcom_pull(key="return_value", task_ids=["transfer_non_native_s3_to_gs"])
+    return len(transferred_files[0])
```

Comment:
@sunank200 Would it make sense to assert the number of files expected here?

Reply:
Yes added logic to list files in s3 and assert: https://github.com/astronomer/apache-airflow-provider-transfers/pull/37/commits/2dc45e0be2cf02f8925ffd530b642a5260882021